locations/spiders/vr_bank.py

```diff
@@ -0,0 +1,188 @@
+                            open_time=open_time,
+                            close_time=close_time,
+                            time_format='%H:%M'
+                        )
+                    else:
+                        continue
+
+        return opening_hours.as_opening_hours()
+
+    def parse_details(self, response):
+        if response.status == 403:
+            yield scrapy.Request(
+                url=response.meta.get('url'),
+                callback=self.parse_details,
+                meta={'url': response.meta.get('url')}
+            )
+
+        name = street = zip = city = phone = website = latitude = longitude = ""
+
+        try:
+            name = response.xpath('//h1[@itemprop="name"]/text()').get()
+        except Exception:
+            pass
+
+        try:
+            street = response.xpath('//span[@itemprop="streetAddress"]/text()').get()
+        except Exception:
+            pass
+        try:
+            zip = response.xpath('//span[@itemprop="postalCode"]/text()').get()
+        except Exception:
+            pass
+
+        try:
+            city = response.xpath('//span[@itemprop="addressLocality"]/text()').get()
+        except Exception:
+            pass
+
+        try:
+            phone = response.xpath('//li[@itemprop="telephone"]/a/span/text()').get()
+        except Exception:
+            pass
+
+        try:
+            website = response.xpath('//li[@itemprop="url"]/a/span/text()').get()
+        except Exception:
+            pass
+
+        try:
+            m = re.search(r'lat&quot;:([-+]?[0-9]*\.?[0-9]*)', response.text)
+            if m:
+                latitude = m.group(1)
+        except Exception:
+            pass
+
+        try:
+            m = re.search(r'lng&quot;:([-+]?[0-9]*\.?[0-9]*)', response.text)
+            if m:
+                longitude = m.group(1)
+        except Exception:
+            pass
+
+        try:
+            hours = response.xpath('//p[@itemprop="openingHoursSpecification"]/text()').getall()
+        except Exception:
+            pass
+
+        properties = {
+            'ref': response.meta.get('url'),
+            'name': name,
+            'city': city,
+            'street': street,
+            'postcode': zip,
+            'phone': phone,
+            'website': website,
+            'lat': latitude,
+            'lon': longitude
+        }
+
+        if hours:
+            properties['opening_hours'] = self.process_hours(hours)
+
+        yield GeojsonPointItem(**properties)
+
+    def get_index(self, response):
+        list = response.xpath('//div[has-class("module module-teaserlist ym-clearfix")]/div/a/@href').getall()
+        for item in list:
+            yield scrapy.Request(
+                url=item,
+                callback=self.parse_details,
+                meta={'url': item}
+            )
+
+    def parse(self, response):
+        index = [
+            "https://www.vr.de/service/filialen-a-z/a.html",
```

Comment:
These requests should be created in the `start_requests()` function, not the `parse()` function.